README.md

```diff
@@ -250,7 +278,23 @@ will take precedence.
 
 Hyrax is available under [the Apache 2.0 license](LICENSE.md).
 
-# Contributing
+# Quick Start for Hyrax development and contributing code back
```

Comment:
Can you remove this section from the PR and merge it into the Sufia Development Guide? (Which I will be copying over to the Hyrax wiki as the Hyrax Development Guide in the coming weeks.) I like to keep the README as small and focused on installation as possible, leaving dev/test concerns in the dev guide.